lib/mayaUsd/ufe/RotationUtils.h

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#pragma once
+
+#include <mayaUsd/mayaUsd.h>
+
+#include <pxr/base/gf/vec3d.h>
+#include <pxr/base/vt/value.h>
+
+#include <maya/MEulerRotation.h>
+#include <ufe/types.h>
+
+PXR_NAMESPACE_USING_DIRECTIVE
+
+namespace MAYAUSD_NS_DEF {
+namespace ufe {
+
+//------------------------------------------------------------------------------
+// Helper functions for dealing with rotations.
+//------------------------------------------------------------------------------
+
+inline double TO_DEG(double a) { return a * 180.0 / 3.141592654; }
```

Comment:
Is it used anywhere else than RotationUtils.cpp? If not, consider moving to an anonymous namespace.